internal/collector/pgbouncer.go

```diff
@@ -49,22 +50,27 @@ func NewConfigForPgBouncerPod(
 func EnablePgBouncerLogging(ctx context.Context,
 	inCluster *v1beta1.PostgresCluster,
 	outConfig *Config,
+	logfile string,
 ) {
 	var spec *v1beta1.InstrumentationLogsSpec
 	if inCluster != nil && inCluster.Spec.Instrumentation != nil {
 		spec = inCluster.Spec.Instrumentation.Logs
 	}
 
 	if OpenTelemetryLogsEnabled(ctx, inCluster) {
-		directory := naming.PGBouncerLogPath
+		directory := filepath.Dir(logfile)
+		create_directory := false
+		if directory != "/tmp" {
+			create_directory = true
+		}
```

Comment:
I ran into an error when I had set the logfile conf to a non `tmp` location -- our collector startup has 
```
` + shell.MakeDirectories(logDir, path.Join(logDir, "receiver"))
```
which would seem to have solved the issue, but I suppose pgbouncer might start up and error out before this. And the pgbouncer container should manage its own logfile location.

Reply:
🤔 In this moment, we could augment the entrypoint.

```go
Command: []string{"sh", "-c", "--",
	mkdirs + `; exec "$@"`, "--",
	"pgbouncer", iniFileAbsolutePath},
```

Reply:
I just re-tested in two scenarios (with OTEL, without OTEL) and even without any special setup of the files, it seemed to work fine. I wonder if the error I saw before was related to the fsGroup not being set.

(Also, when I run `shell.MakeDirectories`, now I get an error about chmod not being allowed.)

Reply:
I've been using GKE for testing this, I want to run some more tests to double-check everything

Reply:
ok, adding back some dir management to the entrypoint, seems fine -- as long as the location exists